modules/checkout/commerce_checkout.info.yml

```diff
@@ -10,3 +10,8 @@ dependencies:
 config_devel:
   install:
     - commerce_checkout.commerce_checkout_flow.default
+    - core.entity_view_display.commerce_order.default.summary
```

Comment:
We don't need the "summary" view mode / form display for orders anymore? Won't be used?